examples/todo-list/test/acceptance/todo-list-todo.acceptance.ts

```diff
@@ -0,0 +1,166 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-todo-list
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {createClientForHandler, expect, supertest} from '@loopback/testlab';
+import {TodoListApplication} from '../../src/application';
+import {Todo, TodoList} from '../../src/models/';
+import {TodoRepository, TodoListRepository} from '../../src/repositories/';
+import {givenTodo, givenTodoList} from '../helpers';
+
+describe('Application', () => {
+  let app: TodoListApplication;
+  let client: supertest.SuperTest<supertest.Test>;
+  let todoRepo: TodoRepository;
+  let todoListRepo: TodoListRepository;
+
+  before(givenRunningApplicationWithCustomConfiguration);
+  after(() => app.stop());
+
+  before(givenTodoRepository);
+  before(givenTodoListRepository);
+  before(() => {
+    client = createClientForHandler(app.requestHandler);
+  });
+
+  beforeEach(async () => {
+    await todoRepo.deleteAll();
+    await todoListRepo.deleteAll();
+  });
+
+  it('creates todo for a todoList', async () => {
+    const todoList = await givenTodoListInstance();
+    const todo = givenTodo();
+    const response = await client
+      .post(`/todo-lists/${todoList.id}/todos`)
+      .send(todo)
+      .expect(200);
+
+    expect(response.body).to.containDeep(todo);
+  });
+
+  it('finds todos for a todoList', async () => {
+    const todoList = await givenTodoListInstance();
+    const notMyTodo = await givenTodoInstance({
+      title: 'someone else does a thing',
+    });
+    const myTodos = [
+      await givenTodoInstanceOfTodoList(todoList.id),
+      await givenTodoInstanceOfTodoList(todoList.id, {
+        title: 'another thing needs doing',
+      }),
+    ];
+    const response = await client
+      .get(`/todo-lists/${todoList.id}/todos`)
+      .send()
+      .expect(200);
+
+    expect(response.body)
+      .to.containDeep(myTodos)
+      .and.not.containEql(notMyTodo.toJSON()); // is this assertion necessary?
+  });
+
+  it('updates todos for a todoList', async () => {
+    const todoList = await givenTodoListInstance();
```

Comment:
In a follow up PR it would be good to move this to a `beforeEach` since it seems repeated in each file. All acceptance tests seem to have a similar repeated line of code at the start of each test.